projects/packages/status/changelog/remove-is-woa-site-at-options-check

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: changed
+Comment: The `is_woa_site` function is now decoupled from the `at_options` option, but this is a WP.com change only
+
+
```

Comment:
There should probably still be something in the package's changelog here. Maybe something like this?
```suggestion
Update logic in `is_woa_site` function for host changes
```

Reply:
That could work, yeah. Thanks for the suggestion